variables.tf

```diff
@@ -0,0 +1,80 @@
+/*
+Module variables
+*/
+
+variable "alb_is_internal" {
+  description = "Determines if the ALB is internal. Default: false"
+  default     = false
+}
+
+variable "alb_name" {
+  description = "The name of the ALB as will show in the AWS EC2 ELB console."
+  default     = "my-alb"
+}
+
+variable "alb_protocols" {
+  description = "A comma delimited list of the protocols the ALB accepts. e.g.: HTTPS"
```

Comment:
Can you make it to accept list natively and join to string, if resource expect string.

Reply:
Sure.